community/detectors/goanywhere_cve_2023_0669/src/test/java/com/google/tsunami/plugins/detectors/goanywhere/Cve20230669DetectorWithCallbackServerTest.java

```diff
@@ -0,0 +1,129 @@
+            new HttpClientModule.Builder().build(),
+            FakePayloadGeneratorModule.builder().setCallbackServer(mockCallbackServer).build(),
+            new Cve20230669DetectorBootstrapModule())
+        .injectMembers(this);
+
+    service =
+        NetworkService.newBuilder()
+            .setNetworkEndpoint(
+                forHostnameAndPort(mockWebServer.getHostName(), mockWebServer.getPort()))
+            .setTransportProtocol(TransportProtocol.TCP)
+            .setSoftware(Software.newBuilder().setName("goanywhere"))
+            .setServiceName("http")
+            .build();
+  }
+
+  @After
+  public void tearDown() throws IOException {
+    mockWebServer.shutdown();
+    mockCallbackServer.shutdown();
+  }
+
+  @Test
+  public void detect_whenVulnerable_returnsVulnerability() throws IOException {
+    mockWebServer.enqueue(new MockResponse().setResponseCode(HttpStatus.OK.code()));
+    mockCallbackServer.enqueue(PayloadTestHelper.generateMockSuccessfulCallbackResponse());
+
+    TargetInfo targetInfo =
+        TargetInfo.newBuilder()
+            .addNetworkEndpoints(
+                forHostnameAndPort(mockWebServer.getHostName(), mockWebServer.getPort()))
+            .build();
+
+    DetectionReportList detectionReports = detector.detect(targetInfo, ImmutableList.of(service));
+
+    assertThat(detectionReports.getDetectionReportsList())
+        .containsExactly(
+            DetectionReport.newBuilder()
+                .setTargetInfo(targetInfo)
+                .setNetworkService(service)
+                .setDetectionTimestamp(
+                    Timestamps.fromMillis(Instant.now(fakeUtcClock).toEpochMilli()))
+                .setDetectionStatus(DetectionStatus.VULNERABILITY_VERIFIED)
+                .setVulnerability(
+                    Vulnerability.newBuilder()
+                        .setMainId(
+                            VulnerabilityId.newBuilder()
+                                .setPublisher("TSUNAMI_COMMUNITY")
+                                .setValue("CVE-2023-0669"))
+                        .addRelatedId(
+                            VulnerabilityId.newBuilder()
+                                .setPublisher("CVE")
+                                .setValue("CVE-2023" + "-0669"))
+                        .setSeverity(Severity.CRITICAL)
+                        .setTitle("CVE-2023-0669: GoAnywhere MFT RCE vulnerability")
+                        .setDescription(
+                            "GoAnywhere MFT suffers from a pre-authentication command injection "
+                                + "vulnerability in the License Response Servlet due to "
+                                + "deserializing an arbitrary attacker-controlled object.All versions prior to 7.1.1 are affected")
+                        .setRecommendation(
+                            "Update GoAnywhere MFT to a version that "
+                                + "provides a fix 7.1.2 or later"))
```

Comment:
```suggestion
                            .setDescription(
                                    "GoAnywhere MFT suffers from a pre-authentication command injection "
                                            + "vulnerability in the License Response Servlet due to deserializing"
                                            + " an arbitrary attacker-controlled object. All versions prior to 7.1.1 are affected.")
                            .setRecommendation(
                                    "Update GoAnywhere MFT to version 7.1.2 or later."))
```
Align Description and Recommendation to the ones in the detector code